src/main/java/dk/kb/ds/cumulus/export/CumulusExport.java

```diff
@@ -68,39 +68,46 @@ public static void main(String[] args) throws Exception {
                 // Get  metadata from Cumulus
                 String id = "ds_" + collection + "_" + record.getFieldValueOrNull("guid");
                 String title = record.getFieldValueOrNull("Titel");
-                String creationDate = record.getFieldValueForNonStringField("Item Creation Date");
-                String created_date = CalendarUtils.getUTCTime(creationDate);
-                if (created_date == null){
-                    created_date_verbatim = creationDate;
-                }
+                String creationDateFromCumulus = record.getFieldValueForNonStringField("Item Creation Date");
+                String created_date = CalendarUtils.getUTCTime(creationDateFromCumulus);
+                created_date_verbatim = getString(created_date_verbatim, creationDateFromCumulus, created_date == null);
```

Comment:
This and elsewhere: Please always use camel case `createdDateVerbatim` instead of underscore, even though the Solr fields use underscore (yes, the conventions differ). Consistency helps when reading code.

It is not too important here as we will hopefully soon shift to a configuration-driven mapping of fields, so this is mainly meant for future code.

Reply:
I thought of changing it but also wanted to see the relationship to the solr-fields. I'll leave it for now since it will be changed soon anyway.